website/static/js/accountSettings.js

```diff
@@ -344,6 +348,48 @@ var ExternalIdentityViewModel = oop.defclass({
     }
 });
 
+var ChangeDefaultStorageLocation = oop.defclass({
+    constructor: function() {
+        this.client = new UserProfileClient();
+        this.profile = ko.observable(new UserProfile());
+        this.locationSelected = ko.observable({'name': ''});
+
+        this.locationSelectedName = ko.computed(function () {
+            return this.locationSelected().name;
+        }, this);
+
+
+        this.client.fetch().done(
+            function(profile) {
+                this.profile(profile);
+                this.locationSelected(this.profile().default_storage_location());
+            }.bind(this)
+        );
+
+
+    },
+    urls: {
+        'change': '/api/v1/profile/region/'
+    },
+    changeDefaultStorageLocation: function() {
+        var request = $osf.ajaxJSON('PUT', this.urls.change, {'data': {'region_id': this.locationSelected().id}});
+        request.done(function() {
+            $osf.growl('Success', 'You have successfully changed you default storage location to <b>' + this.locationSelected().name + '</b>.', 'success');
```

Comment:
"You have successfully changed you default storage location"
should be
"You have successfully changed your default storage location"